wallet/src/types.rs

```diff
@@ -146,19 +148,27 @@ pub struct CreateWalletData<'a> {
     pub account: &'a Account,
 }
 
+#[derive(Copy, Clone, Debug, Serialize, Deserialize)]
+pub enum FeeType {
+    Absolute,
```

Comment:
Should we follow any special naming convention?

```suggestion
    #[serde(rename = "ABSOLUTE")]
    Absolute,
```

```suggestion
    #[serde(rename = "absolute")]
    Absolute,
```

Reply:
which one?

Reply:
I think most literals in the wallet API use uppercase at the moment, so I'd go with uppercase here as well.